Lib/test/test_capi.py

```diff
@@ -999,6 +999,30 @@ async def foo(arg): return await arg  # Py 3.5
             self.assertEqual(ret, 0)
             self.assertEqual(pickle.load(f), {'a': '123x', 'b': '123'})
 
+    def test_int_max_str_digits_config_is_per_interp(self):
```

Comment:
IMO this test belongs to test_int.

In the past, we put random things in test_capi just because it tests the C API. but test_capi became too big.